aws/table_aws_resource_explorer_search.go

```diff
@@ -0,0 +1,150 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/resourceexplorer2"
+	"github.com/aws/aws-sdk-go-v2/service/resourceexplorer2/types"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAWSResourceExplorerSearch(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_resource_explorer_search",
+		Description: "AWS Resource Explorer Search",
+		List: &plugin.ListConfig{
+			Hydrate: awsResourceExplorerSearch,
+			KeyColumns: plugin.KeyColumnSlice{
+				{Name: "query", Require: plugin.Optional, CacheMatch: "exact"},
+				{Name: "region", Require: plugin.Required, CacheMatch: "exact"},
+				{Name: "view_arn", Require: plugin.Optional}, // The view to be used to search resources.
```

Comment:
If a user passes in `view_arn`, they shouldn't need to pass in `view_arn` since we can extract the region info. So can we update the quals to have `region` and `view_arn` be in any `AnyOf` config?